subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/rule/base/BaseActivityScenarioRule.kt

```diff
@@ -0,0 +1,65 @@
+package com.avito.android.rule.base
+
+import android.app.Activity
+import android.content.Intent
+import androidx.annotation.CallSuper
+import androidx.test.core.app.ActivityScenario
+import androidx.test.platform.app.InstrumentationRegistry
+import com.avito.android.rule.SimpleRule
+
+open class BaseActivityScenarioRule<A : Activity>(
+    private val activityClass: Class<A>,
+    private val initialTouchMode: Boolean,
+    private val launchActivity: Boolean,
+) : SimpleRule() {
+
+    private var _scenario: ActivityScenario<A>? = null
```

Comment:
only cosmetics, but there should be a line separator between this var and `val scenario`

I don't remember we use underscore prefix anywhere in code

Reply:
> only cosmetics, but there should be a line separator between this var and val scenario

done

> I don't remember we use underscore prefix anywhere in code

I used a [backing property](https://kotlinlang.org/docs/properties.html#backing-properties) syntax. I can rename if you're against it.

Reply:
no, i'm not